spec/features/inactive_families/admin_views_all_inactive_families_spec.rb

```diff
@@ -0,0 +1,16 @@
+require "rails_helper"
+
+feature "Admin can see all the not active families when in families#index" do
+  context "when click on the 'Not Active' button" do
+    scenario "successfully", js: true do
+      create(:all_true_family)
+      create(:all_false_family)
+
+      visit families_path
+      click_link("Inactive")
```

Comment:
A suggestion to be consistent and use the `id` attribute here like in the other spec

Reply:
I changed all the specs to have the title before reading your comment, I hope it is ok. :)